### PR TITLE
ticket 8

### PR DESCRIPTION
**Pull request by jeeannyy**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -191,12 +184,67 @@ describe('Testing for News app', () => {
...
+    test("200: check an article object has all property", () => {
+      return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then(({body:{ articles }}) => {
+          expect(articles).toHaveLength(12);
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("body");
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: Should not have the body key in article, make sure you don't send the body of the article back

File: __tests__/app.test.js [hunk 2]
```diff
@@ -191,12 +184,67 @@ describe('Testing for News app', () => {
...
+      return request(app)
+        .get(`/api/articles/3`)
+        .expect(200)
+        .then(({body:{ article }}) => {
+          expect(article.comment_count).toEqual(2);
+        });
+    });
+
+    test("200: check the username is in the users table", () => { 
+      return request(app)
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: This test is in the wrong describe block!

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Just a bit of amendment in the test needed - change what I've mentioned and throw it back

Some really great work going on 🎉

File: __tests__/app.test.js (hunk 1 above)
Comment (jeeannyy) [reply __tests__/app.test.js]: Fixed, thanks!

**Review by karlaferry — CHANGES_REQUESTED** [review]
Hi Jeeanne, please see Poonam's comment regarding a test being in the wrong describe block. Can you please move it to the appropriate one? :)

Also, please make sure to only do one ticket per PR so we don't mix things up!

File: __tests__/app.test.js (hunk 2 above)
Comment (jeeannyy) [reply __tests__/app.test.js]: Fixed, thanks!

**Review by Poonam-raj — APPROVED** [review]
All good -merge it in :D 🎉

**Pull request merged**